### ISSUE TITLE
Adding --dropbox-style option to ktfmt (has workaround)

### ISSUE THREAD

**Issue by franvis** [diffplug/spotless#641]
ktfmt uses 2 line indentation by default. The way to move this to 4 is to apply the option `--dropbox-style` when running it. Is there a way to pass this option through spotless?

**Comment by nedtwigg** [comment]
There is not currently, but it would be easy to add.  [Here is how it works](https://github.com/diffplug/spotless/blob/6d43bcccc5e2ba1638c30b0d0fdc8c4f296e4f31/plugin-gradle/src/main/java/com/diffplug/gradle/spotless/JavaExtension.java#L109-L137) for google-java-format's AOSP feature.

An easy workaround is this:

```gradle
ktfmt()
indentWithTabs(2)   // convert leading spaces to tabs at 2 spaces=tab
indentWithSpaces(4) // convert leading tabs to spaces at tab=4 spaces
```

**Comment by franvis** [comment]
Hello @nedtwigg. Thanks for the quick reply. I created a PR to add the option to the step, would you mind taking a look at it? Test are failing but I cannot reproduce the failure locally as well...

**Comment by nedtwigg** [comment]
Published in `plugin-gradle 5.1.0`.
